docs/_source/practical_guides/create_dataset.md

```diff
@@ -435,7 +469,7 @@ record = rg.FeedbackRecord(
 
 
 
-###### Add `responses`
+#### Format `responses`
```

Comment:
I think it is better to explicitly add format metadata and format response vectors too. Explicetely, define everything or nothing, I think.

Reply:
ok I think I'll add all then. I removed that one because I thought it just repeated the information under "Configure the records", but maybe I can make the info there more summarized and add all the relevant sections below.